### PR TITLE
chore: Add Docker latest and nightly tag

### PR DESCRIPTION
**Pull request by pasqualedevita**
Put an `x` into the box if that apply:

- [ ] This PR introduces breaking change.
- [ ] This PR fixes a bug.
- [ ] This PR adds new functionality.
- [X] This PR enhances existing functionality.

### Description of your changes

Add Docker image latest and nightly tag according to https://github.com/antonbabenko/pre-commit-terraform/issues/321

github docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule

### How can we test changes

- tested release on my forked repo
- tested nightly build on my forked repo

### CODE REVIEW

**Comment by yermulnik** [comment]
I was under impression that #321 was about addition of `latest` tag to release builds rather than about nightly builds 😕
I'm not sure how much is the demand from society for the nightly builds since we even don't have any kind of development branch for the cutting edge features not yet included in release.

**Comment by antonbabenko** [comment]
If there is a demand for nightly builds (aka "unstable"), we can build them as `nightly` but not `latest` (aka "stable but unknown version").

**Comment by pasqualedevita** [comment]
thanks boys for suggestions, I tried to fix with:
- `latest` -> created on releases
- `nightly` -> created every night

Probably it's more interesting `latest` other than `nightly`

**Comment by antonbabenko** [comment]
@MaxymVlasov what do you think?

**Review by MaxymVlasov — APPROVED** [review]
LGTM

File: .github/workflows/build-image.yaml [hunk 1]
```diff
@@ -25,7 +27,8 @@ jobs:
       - name: Set tag for image
         run: |
           echo IMAGE_TAG=$([ ${{ github.ref_type }} == 'tag' ] && echo ${{ github.ref_name }} || echo 'latest') >> $GITHUB_ENV
-      - name: Build and Push
+      - name: Build and Push release
+        if: github.event.schedule != '00 00 * * *'
```
[the remark this answers is not in the document]
Comment (pasqualedevita) [reply .github/workflows/build-image.yaml]: thanks, much more clear. done in https://github.com/pasqualedevita/pre-commit-terraform/commit/47e7fc9a9053f686e6127f3e6383faa1032bca62

**Comment by antonbabenko** [comment]
This PR is included in version 1.64.1 :tada:

**Pull request merged**